### PR TITLE
pw-link: update page; pw-*: add pt_BR translation

### PR DESCRIPTION
**Pull request by vitorhcl**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by gutjuri — APPROVED** [review]
Thanks!

File: pages/linux/pw-link.md [hunk 1]
```diff
@@ -3,9 +3,9 @@
 > Manage links between ports in PipeWire.
 > More information: <https://gitlab.freedesktop.org/pipewire/pipewire/-/wikis/Virtual-Devices>.
 
-- List all audio output and input ports:
+- List all audio output and input ports with their ids:
```
Comment (kbdharun) [inline pages/linux/pw-link.md:6]: Welcome to tldr and thanks for your contribution, I would suggest capitalizing ID in the description.

```suggestion
- List all audio output and input ports with their IDs:
```

File: pages/linux/pw-link.md [hunk 2]
```diff
@@ -15,6 +15,10 @@
 
 `pw-link --disconnect {{output_port_name}} {{input_port_name}}`
 
+- List all links with their ids:
```
Comment (kbdharun) [inline pages/linux/pw-link.md:18]:
```suggestion
- List all links with their IDs:
```

File: pages.pt_BR/linux/pw-link.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Criar uma conexão entre uma porta de entrada e uma porta de saída:
+
+`pw-link {{output_port_name}} {{input_port_name}}`
+
+- Desconectar duas portas:
+
+`pw-link --disconnect {{output_port_name}} {{input_port_name}}`
+
+- Listar todas as conexões com seus ids:
```
Comment (kbdharun) [inline pages.pt_BR/linux/pw-link.md:18]:
```suggestion
- Listar todos as saídas e entradas de áudio com seus IDs:

`pw-link --output --input --ids`

- Criar uma conexão entre uma porta de entrada e uma porta de saída:

`pw-link {{output_port_name}} {{input_port_name}}`

- Desconectar duas portas:

`pw-link --disconnect {{output_port_name}} {{input_port_name}}`

- Listar todas as conexões com seus IDs:
```

File: pages.pt_BR/linux/pw-cat.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+> Tocar e gravar arquivos de áudio através do pipewire.
+> Mais informações: <https://fedoraproject.org/wiki/QA:Testcase_PipeWire_PipeWire_CLI>.
+
+- Tocar um arquivo WAV no alvo padrão:
+
+`pw-cat --playback {{caminho/para/arquivo.wav}}`
+
+- Tocar um arquivo WAV com uma qualidade de reamostragem específica (4 por padrão):
+
+`pw-cat --quality {{0..15}} --playback {{caminho/para/arquivo.wav}}`
```
[the remark this answers is not in the document]
Comment (vitorhcl) [reply pages.pt_BR/linux/pw-cat.md:12]: I'm sorry, it is in my other pull request #11734, I accidentally committed to the wrong branch.

File: pages.pt_BR/linux/pw-play.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# pw-play
+
+> Grava arquivos de áudio através do pipewire.
+> Atalho para pw-cat --playback.
+> Mais informações: <https://fedoraproject.org/wiki/QA:Testcase_PipeWire_PipeWire_CLI>.
+
+- Toca um som wav no alvo padrão:
```
Comment (isaacvicente) [inline pages.pt_BR/linux/pw-play.md:7]:
```suggestion
- Toca um som em formato WAV no alvo padrão:
```

File: pages.pt_BR/linux/pw-play.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+`pw-play {{caminho/para/arquivo.wav}}`
+
+- Toca um arquivo de som wav com um volume diferente:
```
Comment (isaacvicente) [inline pages.pt_BR/linux/pw-play.md:11]:
```suggestion
- Toca um arquivo em formato WAV com um volume específico:
```

Comment (isaacvicente) [reply pages.pt_BR/linux/pw-play.md:11]: @vitorhcl Aqui o volume vai de tanto até quanto? Seria interessante deixar isso claro. Temos uma sintaxe para isso.
Por exemplo: `{0.1...0.9}`.

**Review by isaacvicente — COMMENTED** [review]
Fiz algumas sugestões de mudança e comentei numa linha sobre o volume.
Depois de feitas as mudanças, irei fazer o merge.

Muito obrigado pela sua contribuição! :tada:

**Comment by vitorhcl** [comment]
> @vitorhcl Aqui o volume vai de tanto até quanto? Seria interessante deixar isso claro. Temos uma sintaxe para isso. Por exemplo: `{0.1...0.9}`.

@isaacvicente @sebastiaanspeck @kbdharun @gutjuri this is a valid question in the English version of the page as well.

There isn't a limited range for the volume, you can set for example the volume to 125% using --volume 1.25, so what if we made it a bit more clear by changing the example and the description?

File: pages.pt_BR/linux/pw-record.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+> Grava arquivos de áudio através do pipewire.
+> Atalho para pw-cat --record.
+> Mais informações: <https://fedoraproject.org/wiki/QA:Testcase_PipeWire_PipeWire_CLI>.
+
+- Faz uma gravação usando o alvo padrão:
+
+`pw-record {{caminho/para/arquivo.wav}}`
+
+- Faz uma gravação com um volume diferente:
```
Comment (vitorhcl) [inline pages.pt_BR/linux/pw-record.md:11]:
```suggestion
- Faz uma gravação com o volume em 125%:
```

**Comment by vitorhcl** [comment]
> > > @vitorhcl Aqui o volume vai de tanto até quanto? Seria interessante deixar isso claro. Temos uma sintaxe para isso. Por exemplo: `{0.1...0.9}`.
> > 
> > 
> > @isaacvicente @sebastiaanspeck @kbdharun @gutjuri this is a valid question in the English version of the page as well.
> > There isn't a limited range for the volume, you can set for example the volume to 125% using --volume 1.25, so what if we made it a bit more clear by changing the example and the description?
> 
> The default volume is 1.000, so what would be the point of setting the volume higher than that?

I think it is a useful feature when the audio volume of the file is too low.

**Comment by isaacvicente** [comment]
I think everything seems fine here @sebastiaanspeck.

**Review by isaacvicente — APPROVED** [review]
LGTM :tada:

**Review by kbdharun — APPROVED** [review]

**Pull request merged**